examples/policies/README.md

```diff
@@ -0,0 +1,31 @@
+# Policies examples
+
+You can start example configuration with example policy by runing this from the APIcast root directory:
```

Comment:
I think that in order to be able to follow the example, we need to specify here what's the policy being loaded in the example configuration file and what it does.

Reply:
Better now?

Reply:
:+1: